backend/src/utils/constants.ts

```diff
@@ -137,9 +137,6 @@ export const DEFAULT_NOTEBOOK_SIZES: NotebookSize[] = [
   },
 ];
 
-export const imageUrlRegex =
```

Comment:
Duplicate and unused version of the regexp that was deleted on the frontend for not being deadcode.